test/Microsoft.AspNetCore.Razor.Evolution.Test/DefaultDirectiveIRPassTest.cs

```diff
@@ -226,7 +193,7 @@ private static DocumentIRNode Lower(RazorCodeDocument codeDocument, RazorEngine
             Assert.NotNull(irDocument);
 
             // These tests depend on the document->namespace->class structure.
-            irDocument = new DefaultDocumentClassifierPass() { Engine = engine, }.Execute(codeDocument, irDocument);
+            new DefaultDocumentClassifierPass() { Engine = engine, }.Execute(codeDocument, irDocument);
```

Comment:
If this still needed? You should be able to execute phases up to the directive phase and then stop.